api/v1/jats/PKPJatsController.php

```diff
@@ -0,0 +1,213 @@
+<?php
+
+/**
+ * @file api/v1/jats/PKPJatsController.php
```

Comment:
If this is going to take care of the `submissions/{submissionId}/publications/{publicationId}/jats` endpoint, then it should be in `api/v1/submissions` or `api/v1/submissions/jats`. I think you can direct execution there by adding the new handler to `api/v1/submissions/index.php`; then you wouldn't need to e.g. list the new operations in the `PKPSubmissionController`.

Reply:
I think I did add the reference to the `PKPJatsController` API Controller in the `submissions/index.php`, and removed all references/endpoints regarding JATS from the `PKPSubmissionController`. 

Are you referring to something else here?

Reply:
The operations in `PKPJatsController` [are still listed](https://github.com/pkp/pkp-lib/pull/9536/files#diff-3673647c7b54136c9bd00654b29a449ee67288a533f1f08e14452b4777ca01edR106) in `PKPSubmissionController`; can that be moved to `PKPJatsController`?

Reply:
Removed from `PKPSubmissionController`. Added the `$this->addPolicy(new PublicationWritePolicy($request, $args, $roleAssignments));` in the `PKPJatsController` instead of the `PublicationAccessPolicy`